x/oracle/keeper/keeper_test.go

```diff
@@ -24,7 +24,8 @@ import (
 )
 
 const (
-	exchangeRate string = umeeapp.DisplayDenom
+	exchangeRate      string = umeeapp.DisplayDenom
```

Comment:
Clarify variable name - current one implies a number instead of a string

Reply:
agree